src/sentry/eventstream/base.py

```diff
@@ -71,20 +72,24 @@ def _dispatch_post_process_group_task(
         primary_hash: Optional[str],
         skip_consume: bool = False,
         group_states: Optional[GroupStates] = None,
+        group_category: Optional[GroupCategory] = GroupCategory.ERROR,
```

Comment:
Doesn't need to be optional since you have default value which is a GroupCategory.

Reply:
Don't events without any groups still come through this function? I'm not sure that group category makes a ton of sense here when there is no group.

Reply:
That's a reasonable point. I'm not sure if we should have a separate queue for issueless events, but maybe for now it does make sense to just rely on the event type

Reply:
Either way though, we're going to have to pass something through here. By the time we get to https://github.com/getsentry/sentry/blob/28cbc543bbff5ca02972772b8c32c211d0fff199/src/sentry/eventstream/kafka/postprocessworker.py#L68 we already only have event ids, and I think it's preferable to not load things from the cache at this point?